tcms/issuetracker/base.py

```diff
@@ -57,7 +57,9 @@ def _report_comment(self, execution):  # pylint: disable=no-self-use
         comment = "Filed from execution %s\n\n" % execution.get_full_url()
         comment += "**Product:**\n%s\n\n" % execution.run.plan.product.name
         comment += "**Component(s):**\n%s\n\n" % \
-                   execution.case.component.values_list('name', flat=True)
+                   ' '.join([str(i) for i in
```

Comment:
use `', '` for delimiter

Reply:
We have a linter which doesn' tlike list comprehensions so don't use them. 

`values_list()` should be returning a list so you should be able to join that directly without using a list comprehension.